ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/common/util/SyncCommitteeUtil.java

```diff
@@ -172,6 +173,13 @@ public boolean isSyncCommitteeAggregator(final BLSSignature signature) {
     return assignments != null ? assignments.getAssignedSubcommittees() : emptySet();
   }
 
+  public Set<Integer> getCommitteeIndices(
+      final BeaconState state, final UInt64 epoch, final UInt64 validatorIndex) {
+    final SyncSubcommitteeAssignments assignments =
+        getSyncSubcommittees(state, epoch).get(validatorIndex);
+    return assignments != null ? assignments.getCommitteeIndices() : emptySet();
```

Comment:
Could be simplified to: 
```suggestion
    final SyncSubcommitteeAssignments assignments =
        getSyncSubcommittees(state, epoch).getOrDefault(validatorIndex, SyncSubcommitteeAssignments.NONE);
    return assignments.getCommitteeIndices();
```